<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  
  
  <meta name="generator" content="Wowchemy 5.0.0-beta.2 for Hugo">
  

  

  
  

  
  
  
  
  
    
    
    
  
  

  

  
  
  
    
  
  <meta name="description" content="One of the major methods to identify microbial community composition, to unravel microbial population dynamics, and to explore microbial diversity in environmental samples is DNA- or RNA-based 16S rRNA (gene) amplicon sequencing. Subsequent bioinformatics analyses are required to extract valuable information from the high-throughput sequencing approach. However, manifold bioinformatics tools complicate their choice and might cause differences in data interpretation, making the selection of the pipeline a crucial step. Here, we compared the performance of most widely used 16S rRNA gene amplicon sequencing analysis tools (i.e. Mothur, QIIME1, QIIME2, and MEGAN) using mock datasets and environmental samples from contrasting terrestrial and freshwater sites. Our results showed that QIIME2 outcompeted all other investigated tools in sequence recovery (&gt;10 times less false positives), taxonomic assignments (&gt;22% better F-score) and diversity estimates (&gt;5% better assessment), while there was still room for improvement e.g. imperfect sequence recovery (recall up to 87%) or detection of additional false sequences (precision up to 72%). Furthermore, we found that microbial diversity estimates and highest abundant taxa varied among analysis pipelines (i.e. only one in five genera was shared among all analysis tools) when analyzing environmental samples, which might skew biological conclusions. Our findings were subsequently implemented in a high-performance computing conformant workflow following the FAIR (Findable, Accessible, Interoperable, and Re-usable) principle, allowing reproducible 16S rRNA gene amplicon sequence analysis starting from raw sequence files. Our presented workflow can be utilized for future studies, thereby facilitating the analysis of high-throughput DNA- or RNA-based 16S rRNA (gene) sequencing data substantially.">

  
  <link rel="alternate" hreflang="en-us" href="https://apeltzer.github.io/publication/17-ampliseq/">

  







  




  
  
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  

  
  
  
  <meta name="theme-color" content="#1565c0">
  

  
  

  
  
  
  
    
    
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous" media="print" onload="this.media='all'">

    
    
    
      
    
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css" integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js" integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media="print" onload="this.media='all'">
    
  

  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.36a8fd48d188121ebf05a62073583db8.css">

  




  

  


  
  

  

  
  <link rel="manifest" href="/index.webmanifest">
  

  <link rel="icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_180x180_fill_lanczos_center_2.png">

  <link rel="canonical" href="https://apeltzer.github.io/publication/17-ampliseq/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="Personal Homepage of Alex Peltzer">
  <meta property="og:url" content="https://apeltzer.github.io/publication/17-ampliseq/">
  <meta property="og:title" content="Interpretations of microbial community studies are biased by the selected 16S rRNA gene amplicon sequencing pipeline. | Personal Homepage of Alex Peltzer">
  <meta property="og:description" content="One of the major methods to identify microbial community composition, to unravel microbial population dynamics, and to explore microbial diversity in environmental samples is DNA- or RNA-based 16S rRNA (gene) amplicon sequencing. Subsequent bioinformatics analyses are required to extract valuable information from the high-throughput sequencing approach. However, manifold bioinformatics tools complicate their choice and might cause differences in data interpretation, making the selection of the pipeline a crucial step. Here, we compared the performance of most widely used 16S rRNA gene amplicon sequencing analysis tools (i.e. Mothur, QIIME1, QIIME2, and MEGAN) using mock datasets and environmental samples from contrasting terrestrial and freshwater sites. Our results showed that QIIME2 outcompeted all other investigated tools in sequence recovery (&gt;10 times less false positives), taxonomic assignments (&gt;22% better F-score) and diversity estimates (&gt;5% better assessment), while there was still room for improvement e.g. imperfect sequence recovery (recall up to 87%) or detection of additional false sequences (precision up to 72%). Furthermore, we found that microbial diversity estimates and highest abundant taxa varied among analysis pipelines (i.e. only one in five genera was shared among all analysis tools) when analyzing environmental samples, which might skew biological conclusions. Our findings were subsequently implemented in a high-performance computing conformant workflow following the FAIR (Findable, Accessible, Interoperable, and Re-usable) principle, allowing reproducible 16S rRNA gene amplicon sequence analysis starting from raw sequence files. Our presented workflow can be utilized for future studies, thereby facilitating the analysis of high-throughput DNA- or RNA-based 16S rRNA (gene) sequencing data substantially."><meta property="og:image" content="https://apeltzer.github.io/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="https://apeltzer.github.io/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2020-10-23T00:00:00&#43;02:00">
    
    <meta property="article:modified_time" content="2020-10-23T00:00:00&#43;02:00">
  

  


    









<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://apeltzer.github.io/publication/17-ampliseq/"
  },
  "headline": "Interpretations of microbial community studies are biased by the selected 16S rRNA gene amplicon sequencing pipeline.",
  
  "datePublished": "2020-10-23T00:00:00+02:00",
  "dateModified": "2020-10-23T00:00:00+02:00",
  
  "author": {
    "@type": "Person",
    "name": "Straub D"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "Personal Homepage of Alex Peltzer",
    "logo": {
      "@type": "ImageObject",
      "url": "https://apeltzer.github.io/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png"
    }
  },
  "description": "One of the major methods to identify microbial community composition, to unravel microbial population dynamics, and to explore microbial diversity in environmental samples is DNA- or RNA-based 16S rRNA (gene) amplicon sequencing. Subsequent bioinformatics analyses are required to extract valuable information from the high-throughput sequencing approach. However, manifold bioinformatics tools complicate their choice and might cause differences in data interpretation, making the selection of the pipeline a crucial step. Here, we compared the performance of most widely used 16S rRNA gene amplicon sequencing analysis tools (i.e. Mothur, QIIME1, QIIME2, and MEGAN) using mock datasets and environmental samples from contrasting terrestrial and freshwater sites. Our results showed that QIIME2 outcompeted all other investigated tools in sequence recovery (\u003e10 times less false positives), taxonomic assignments (\u003e22% better F-score) and diversity estimates (\u003e5% better assessment), while there was still room for improvement e.g. imperfect sequence recovery (recall up to 87%) or detection of additional false sequences (precision up to 72%). Furthermore, we found that microbial diversity estimates and highest abundant taxa varied among analysis pipelines (i.e. only one in five genera was shared among all analysis tools) when analyzing environmental samples, which might skew biological conclusions. Our findings were subsequently implemented in a high-performance computing conformant workflow following the FAIR (Findable, Accessible, Interoperable, and Re-usable) principle, allowing reproducible 16S rRNA gene amplicon sequence analysis starting from raw sequence files. Our presented workflow can be utilized for future studies, thereby facilitating the analysis of high-throughput DNA- or RNA-based 16S rRNA (gene) sequencing data substantially."
}
</script>

  

  


  


  





  <title>Interpretations of microbial community studies are biased by the selected 16S rRNA gene amplicon sequencing pipeline. | Personal Homepage of Alex Peltzer</title>

</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="page-wrapper  ">

  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.8446300de3057dfecd2dbb1103d0e171.js"></script>

  

<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header">
    












<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container-xl">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">Personal Homepage of Alex Peltzer</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">Personal Homepage of Alex Peltzer</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#posts"><span>Posts</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects"><span>Projects</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#talks"><span>Talks</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

      
      
        
      

      
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
          <i class="fas fa-moon" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>


  </div>

  <div class="page-body">
    








<div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>Interpretations of microbial community studies are biased by the selected 16S rRNA gene amplicon sequencing pipeline.</h1>

  

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Straub D</span>, <span >
      Blackwell N</span>, <span >
      Langarica Fuentes A</span>, <span >
      Peltzer A</span>, <span >
      Nahnsen S</span>, <span >
      Kleindienst S</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October 2020
  </span>
  

  

  

  
  
  
  
  

  
  

</div>

    




<div class="btn-links mb-3">
  
  








  
















  
  
    
  
<a class="btn btn-outline-primary btn-page-header" href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.550420/full" target="_blank" rel="noopener">
  Source Document
</a>




</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">One of the major methods to identify microbial community composition, to unravel microbial population dynamics, and to explore microbial diversity in environmental samples is DNA- or RNA-based 16S rRNA (gene) amplicon sequencing. Subsequent bioinformatics analyses are required to extract valuable information from the high-throughput sequencing approach. However, manifold bioinformatics tools complicate their choice and might cause differences in data interpretation, making the selection of the pipeline a crucial step. Here, we compared the performance of most widely used 16S rRNA gene amplicon sequencing analysis tools (i.e. Mothur, QIIME1, QIIME2, and MEGAN) using mock datasets and environmental samples from contrasting terrestrial and freshwater sites. Our results showed that QIIME2 outcompeted all other investigated tools in sequence recovery (&gt;10 times less false positives), taxonomic assignments (&gt;22% better F-score) and diversity estimates (&gt;5% better assessment), while there was still room for improvement e.g. imperfect sequence recovery (recall up to 87%) or detection of additional false sequences (precision up to 72%). Furthermore, we found that microbial diversity estimates and highest abundant taxa varied among analysis pipelines (i.e. only one in five genera was shared among all analysis tools) when analyzing environmental samples, which might skew biological conclusions. Our findings were subsequently implemented in a high-performance computing conformant workflow following the FAIR (Findable, Accessible, Interoperable, and Re-usable) principle, allowing reproducible 16S rRNA gene amplicon sequence analysis starting from raw sequence files. Our presented workflow can be utilized for future studies, thereby facilitating the analysis of high-throughput DNA- or RNA-based 16S rRNA (gene) sequencing data substantially.</p>
    

    
    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            <a href="/publication/#2">
              Journal article
            </a>
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9">In <em>Frontiers in Microbiology</em></div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style"></div>

    






<div class="article-tags">
  
  <a class="badge badge-light" href="/tag/amplicon/">amplicon</a>
  
  <a class="badge badge-light" href="/tag/16s/">16S</a>
  
  <a class="badge badge-light" href="/tag/rrna/">rRNA</a>
  
  <a class="badge badge-light" href="/tag/amplicon-sequencing/">amplicon sequencing</a>
  
  <a class="badge badge-light" href="/tag/environmental-omics/">environmental omics</a>
  
</div>



<div class="share-box" aria-hidden="true">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=https://apeltzer.github.io/publication/17-ampliseq/&amp;text=Interpretations%20of%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20gene%20amplicon%20sequencing%20pipeline." target="_blank" rel="noopener" class="share-btn-twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=https://apeltzer.github.io/publication/17-ampliseq/&amp;t=Interpretations%20of%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20gene%20amplicon%20sequencing%20pipeline." target="_blank" rel="noopener" class="share-btn-facebook">
          <i class="fab fa-facebook"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Interpretations%20of%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20gene%20amplicon%20sequencing%20pipeline.&amp;body=https://apeltzer.github.io/publication/17-ampliseq/" target="_blank" rel="noopener" class="share-btn-email">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=https://apeltzer.github.io/publication/17-ampliseq/&amp;title=Interpretations%20of%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20gene%20amplicon%20sequencing%20pipeline." target="_blank" rel="noopener" class="share-btn-linkedin">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="whatsapp://send?text=Interpretations%20of%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20gene%20amplicon%20sequencing%20pipeline.%20https://apeltzer.github.io/publication/17-ampliseq/" target="_blank" rel="noopener" class="share-btn-whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=https://apeltzer.github.io/publication/17-ampliseq/&amp;title=Interpretations%20of%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20gene%20amplicon%20sequencing%20pipeline." target="_blank" rel="noopener" class="share-btn-weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>











  
  
    




  
    




  
    




  
    




  
    




  
    




  














  
  
  <div class="article-widget content-widget-hr">
    <h3>Related</h3>
    <ul>
      
      <li><a href="/publication/13-ampliseq/">Interpretations of microbial community studies are biased by the selected 16S rRNA gene amplicon sequencing pipeline.</a></li>
      
    </ul>
  </div>
  





  </div>
</div>
  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">
  

  <p class="powered-by">
    2021
  </p>

  
  






  <p class="powered-by">
    
    
    
    Published with
    <a href="https://wowchemy.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Wowchemy</a>  —
    the free, <a href="https://github.com/wowchemy/wowchemy-hugo-modules" target="_blank" rel="noopener">
    open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>

      
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      

      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js" integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin="anonymous"></script>

      

      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.js" integrity="sha512-SeiQaaDh73yrb56sTW/RgVdi/mMqNeM2oBwubFHagc5BkixSpP1fvqF47mKzPGWYSSy4RwbBunrJBQ4Co8fRWA==" crossorigin="anonymous"></script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
        <div class="search-hit-content">
          <div class="search-hit-name">
            <a href="{{relpermalink}}">{{title}}</a>
            <div class="article-metadata search-hit-type">{{type}}</div>
            <p class="search-hit-description">{{snippet}}</p>
          </div>
        </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    
    

    
    
    

    
    

    
    

    
    
      
      
      
      
      
      
      
    

    
    
    
    
    
    
    
    
    
    
      
      
    
    
    <script src="/en/js/wowchemy.min.840ff069a6d25db681d120b4f843b863.js"></script>

    






</body>
</html>
